### PR TITLE
German translations

### PR DESCRIPTION
**Pull request by Schnoffel**
Here another try with a new PR.

### CODE REVIEW

File: translations/monero-core_de.ts [hunk 1]
```diff
@@ -1086,7 +1086,7 @@
     <message>
         <location filename="../pages/Sign.qml" line="282"/>
         <source>Verify a message or file signature from an address:</source>
-        <translation>Nachricht oder Datei von einer Adresse verifizieren</translation>
+        <translation>Nachricht oder Datei einer Adresse verifizieren</translation>
```
Comment (erciccione) [inline translations/monero-core_de.ts]: Forgotten `:` at the end

File: translations/monero-core_de.ts [hunk 2]
```diff
@@ -1866,13 +1873,13 @@ Ringsize: </source>
...
-        <translation>Erstelle ein Passwort für dein Wallet</translation>
+        <translation>Erstelle ein Passwort für Dein Wallet</translation>
     </message>
     <message>
         <location filename="../wizard/WizardPassword.qml" line="127"/>
         <source> &lt;br&gt;Note: this password cannot be recovered. If you forget it then the wallet will have to be restored from its 25 word mnemonic seed.&lt;br/&gt;&lt;br/&gt;
                         &lt;b&gt;Enter a strong password&lt;/b&gt; (using letters, numbers, and/or symbols):</source>
-        <translation> &lt;br&gt;Beachte: Das Passwort kann nicht wiederhergestellt werden und wenn du es vergisst, kannst Du nur Zugriff auf dein Wallet bekommen, indem du den&lt;br/&gt;&lt;br/&gt;
+        <translation> &lt;br&gt;Beachte: Das Passwort kann nicht wiederhergestellt werden. Wenn Du es vergisst, kannst Du nur Zugriff auf Dein Wallet bekommen, indem Du den&lt;br/&gt;&lt;br/&gt;
                         aus 25 Wörtern bestehenden mnemonischen Code eingibst, der Dir bei der Einrichtung angezeigt wurde. Das Passwort schützt das Wallet und jede damit verbundene Aktion. Verwende also ein sicheres Passwort.</translation>
```
Comment (erciccione) [inline translations/monero-core_de.ts]: used `.` instead of `:`, as in the source string

File: translations/monero-core_de.ts [hunk 3]
```diff
@@ -1955,12 +1962,12 @@ Ringsize: </source>
...
         <location filename="../main.qml" line="718"/>
         <source>Payment check</source>
-        <translation>Zahlungscheck</translation>
+        <translation>Zahlungsüberprüfung</translation>
     </message>
     <message>
         <location filename="../main.qml" line="723"/>
         <source>This address received %1 monero, but the transaction is not yet mined</source>
-        <translation>Diese Adresse hat %1 Monero empfangen, aber die Transaktion wurde noch nicht geschürft.</translation>
+        <translation>Diese Adresse hat %1 Monero empfangen, aber die Transaktion wurde noch nicht gemined.</translation>
```
Comment (erciccione) [inline translations/monero-core_de.ts]: the `.` at the end is not in the source string

**Comment by Schnoffel** [comment]
I cannot change that because I would have to merge it again into one commit unfortunately.

**Comment by MaxXor** [comment]
You can change that, just do it the same way you opened up this PR if you want to avoid the squash.

**Comment by leonklingele** [comment]
I'll provide a step-by-step tutorial for you to follow @Schnoffel:

```sh
$ git clone git@github.com:Schnoffel/monero-gui.git
$ cd monero-gui
# Now fix the three punctuation issues and save the file
$ git commit --amend --no-edit
$ git push -f
```

**Comment by Schnoffel** [comment]
@erciccione : Please for the next PR as the faults were not caused by this PR : )

**Comment by Schnoffel** [comment]
@MaxXor : I am really worked up now about your thumbs down. I am abroad for a few weeks and have no time for this, this and this change. The PR for now is finished and correct. So I do not catch the point why the hell the work which was done and finished six weeks ago is not going to be just merged into the core. I already said that I will do a new PR when I find time to do a new lot of translations.

When I decided to contribute to monero I did not decide to do this all the day. I have much work to do and it was just like a present to the community because I like the monero project very much. But you don't take what get but you complain that I do not contribute more than I already have done.

As this already took to much time for nothing and you are not happy with that I have already done I close the PR. So good bye.

**Comment by erciccione** [comment]
@Schnoffel It's not important who did the "mistake", the important is that you have the possibility to fix it. An open source project works because a bunch of selfless people work together to create something great. Nobody gave you a time limit of any sort, but instead you received a lot of support, also after refusing to learn how to contribute properly. @leonklingele even made you a step-by-step tutorial. All these people spent their free time to help you, and now you are closing this pr, throwing everybody's work in the bin, because of a thumb down...Really? Goodbye, you will not be missed

**Comment by Schnoffel** [comment]
@erciccione : Then fix it yourself and do not annoy other contributers. It works because people like me contribute and such projects die or get forked because of people like you. People who do not respect the work of others and always think they know it better. The goal of the PR was never to fix 100 % of all mistakes the previous translation had. The goal was just to improve the translation what I did. What others already have done. But you are not happy with it, instead you want to make a infinite story of it.

I am also not throwing work into the bin. You are the one who stopped this PR do be finally finished after almost two months. You're also free to take the work and make an own PR.

**Pull request closed without merging**